packages/scripts/src/commands/app-data/convert/index.ts

```diff
@@ -24,10 +24,16 @@ import { getActiveDeployment } from "../../deployment/get";
 const program = new Command("convert");
 interface IProgramOptions {
```

Comment:
Refactoring to expose existing commands system to tasks (to save having to duplicate code)